src/osha/oira/statistics/utils.py

```diff
@@ -92,6 +95,7 @@ def update_tool(self, country=None):
                 ),
                 sqlalchemy.func.count(SurveySession.id),
             )
+            .filter(Survey.published_date > latest_published_date)
```

Comment:
This implicitly adds a filter on the published date, right?
Maybe line 100 `.filter(Survey.published)` can be removed?

Reply:
You're probably right. I wonder whether there could be a case where `published` is false but `published_date` is still set because the tool was previously published... I think I'd rather keep both filters to be sure.

Reply:
The column `published` in the session is not a boolean but a date.
```
    published = schema.Column(
        types.DateTime,
        nullable=True,
        default=None,
    )
```
I might not fully grasp how it works but the idea behind that is that you do not want two columns to express that concept (e.g. `published` and `published_date`, `archived` and `archived_date` and so on).

Reply:
Ah, but it filters on `Survey.published`, not `SurveySession.published`.

(Unfortunately in this case `Survey` is an alias of `SurveyStatistics` because both `osha.oira.statistics.model` and `osha.oira.client.model` have a class of that name. Some cleanup could be done there.)